frontend/src/App.tsx

```diff
@@ -45,7 +45,11 @@ import { SessionEventDispatcher } from "src/lib/SessionEventDispatcher"
 import {
   setCookie,
   hashString,
-  isEmbeddedInIFrame,
+  hasEmbedUrlParamSetToTrue,
+  hasEmbedUrlParamSetToNoToolBar,
+  hasEmbedUrlParamSetToNoFooter,
+  hasEmbedUrlParamSetToNoPadding,
+  hasEmbedUrlParamSetToNoScroll,
```

Comment:
nit: I would consider a bit different naming:
* `isEmbed`
* `isToolbarEnabled` or `toolbarEnabled`
* `isFooterEnabled` or `footerEnabled`
* `isPaddingEnabled` or `paddingEnabled`
* `isScrollingEnabled` or `scrollingEnabled`

Reply:
The PR is basically to ask more specific questions related to spec, because it's not ready, hence I asked @jrieke to look at the feature-demo. Also it's good to get early insights into the code from other people, so thank you for the review 🙇 

> In this PR we are not touching the embedded option and this is by design, right?
Actually, I didn't see `embedded` query param, anywhere in the code, there is an `embedded` css class, I asked @jrieke if we're sure that the core is responsible for `embedded` param 

> The number of available embed params, their names and functionality is agreed, correct? 
No, it's not specified exactly yet, I hope this PR and its feature demo speeds things up 🙂